Client/src/services/apiService.ts

```diff
@@ -0,0 +1,31 @@
+const BASE_URL = 'http://localhost:3000'; //TODO Add to .env
+
+type ApiService = {
+  profile: Function,
+  updateProfile: Function,
+}
+const apiService: any = {};
+
+apiService.profile = function () {
+  return fetch(`${BASE_URL}/profile`, {
+    method: 'GET',
+    credentials: 'include',
+    headers: { "Access-Control-Allow-Origin": '*', 'Content-Type': 'application/json' }
+  })
```

Comment:
well done josep, remember to add a new query to retrieve avatars and keep the aPi service type in a types folder

Reply:
yep keep up the good work josep!